daml-lf/language/BUILD.bazel

```diff
@@ -12,6 +12,7 @@ da_scala_library(
     name = "language",
     srcs = glob(["src/main/**/*.scala"]),
     scalacopts = lf_scalacopts,
+    silent_annotations = True,
```

Comment:
Is this intended?

Reply:
This goes with the `import com.github.ghik.silencer.silent` you refer under and the annotation `@silent` line 723. 
I drop everything as it is not really necessary.

Reply:
I see. Thanks for the explanation.